vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/verdict.rb

```diff
@@ -0,0 +1,26 @@
+module SecureRooms
+
+  module AccessRules
+
+    class Verdict
+
+      attr_accessor :reason, :result_code
+
+      def initialize(result_code, reason = nil)
+        @result_code = result_code
+        @reason = reason
+      end
+
+      def pass?
+        @result_code == :pass
+      end
+
+      def has_result_code?(code)
```

Comment:
it's only used in the specs... it's not _exactly_ code just for testing purposes but it does kinda feel like it 😳 what's your opinion on keeping it in for just that readability?